src/style.css

```diff
@@ -0,0 +1,102 @@
+* {
+  padding: 0;
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 600;
+}
+
+html {
+  display: block;
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+}
+
+.search-form {
+  display: flex;
+  justify-content: center;
+  margin: 20px auto 0;
+  width: 100%;
+  height: 70px;
+}
+
+.search-form_small {
```

Comment:
Like Radek mentioned before the modifier classes should be `--small` or `--big` rather than `_small`. So your modifier classes should be `.search-form--small`. Please change it for all elements that use modifiers in .css file & html :)